index.js

```diff
@@ -0,0 +1,94 @@
+const URL = 'https://trektravel.herokuapp.com/trips/';
+
+$(document).ready(function(){
+
+  let numTrips = 10;
+
+  let tripTitle = function tripTitle(response, id){
+    let name = response.name || 'untitled';
+    let weeks = response.weeks || '0';
+    return `<li class="trip no-show" id="${response.id}">
+      <div class="row">
+        <h3 class="small-8 medium-9 columns">${name}</h3>
+        <h5 class="trip-weeks">${weeks} weeks</h5>
+      </div>
+    </li>`;
+  };
+
+  let tripAbout = function tripAbout(response, id){
+    let continent = response.continent || 'unavailable';
+    let category = response.category || 'unavailable';
+    let cost = response.cost || '0';
+    let about = response.about || 'no information currently available' ;
+    return `<div class="about-trip">
+      <div class="trip-info small-12 medium-6 columns">
+        <h5>
+          <strong>continent: </strong> ${continent}
+        </h5>
+      </div>
+      <div class="trip-info small-12 medium-6 columns">
+        <h5>
+          <strong>category: </strong> ${category}
+        </h5>
+      </div>
+      <div class="trip-info small-12 columns">
+        <p class="trip-about">
+          <em>${about}</em>
+        </p>
+        <a class="book-btn button no-collapse small-12 medium-6 large-3 columns">Book Now: $${cost.toFixed(2)}</a>
+        <form class="hidden book-form no-collapse small-12 medium-8 columns row" id="book-form-${id}">
+          <label for="email" class="no-collapse small-12 columns"><p><em>Enter your email address to reserve your spot now!</em></p></label>
+          <input type="text" class="no-collapse small-12 medium-9 columns" name="email" placeholder="example@test.com" />
+
+          <input type="submit" class="no-collapse button small-12 medium-3 columns" />
+        </form>
+      </div>
+    </div>`;
+  };
+
+  $('#hero').on('click', '.button', function(e){
+    $.get(URL, function(response) {
+      for(let i = 0; i < response.length; i++) {
+        $('#trip-list').append(tripTitle(response[i], i));
+      }
+    });
+
+    $('main > section').toggleClass('hidden');
+  });
+
+  $('#trip-list').on('click', function(e){
+    console.log(e);
+    if (!e.target.className.includes('no-collapse')) {
+      let id = e.target.closest('.trip').id;
```

Comment:
These two lines where you figure out what trip was clicked end up being pretty complicated. This means it's hard to read, and is also usually a good indicator that they're fragile - that is, a change somewhere else is likely to break them unexpectedly.

Instead, you might register the click handler like this:

```javascript
$('#trip-list').on('click', 'li.trip', function(e) {
```

Then inside the callback, `$(this)` will always refer to the `<li>` that was clicked, which should make it simpler to pull out the ID.